packages/design-system/src/components/combobox.tsx

```diff
@@ -52,18 +52,36 @@ export const ComboboxTextField = forwardRef(ComboboxTextFieldBase);
 
 ComboboxTextField.displayName = "ComboboxTextField";
 
-const Listbox = styled("ul", panelStyles, {
-  padding: 0,
+const Listbox = styled("ul", {
   margin: 0,
+  padding: 0,
   overflow: "auto",
-  // @todo need some non-hardcoded value
-  maxHeight: 400,
-  minWidth: 230,
+  minWidth: 124,
+  maxHeight: 190,
```

Comment:
Todo is removed but it wasn't done since we still use hard coded values. Also new values will probably break other parts of UI.

Reply:
forgotten? what can we do to avoid hardcoding those width/height values? maybe those should be on the consumer side?